extensions/dou-tu/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "dou-tu",
+  "title": "斗图",
+  "description": "An extension for easy retrieval of web emoji images while chatting.(上班时间斗个图🌟)",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.